sig-windows/annual-report-2020

```diff
@@ -0,0 +1,85 @@
+# Kubernetes Community Group Annual Reports Template 
+
+Checklist 
+[ ] Read about the process here (https://git.k8s.io/community/committee-steering/governance/annual-reports.md#reporting-process) 
+[x] Copy this template into a new document and share with your mailing list/slack channel/meeting on whatever platform (gdocs, hackmd, etc.) that the team prefers.
+[x] Remove sections that are not applicable (example: if you are a working group, delete the special interest group questions)
+[ ] Pick graphs from Devstats to pull supporting data for your responses. 
+[ ] Schedule a time with your Steering liaison and other Chairs, TLs, and Organizers of your group to check-in on your roles as Chair or Working Group Organizer. If anyone would rather meet 1:1, please have them reach out to the liaison directly, we are happy to. We’d like to talk about: challenges, wins, things you didn’t know before but wish you did, want to continue in the role or help finding a replacement; and lastly any feedback you have for us as a body and how we can help you succeed and feel comfortable in these leadership roles. 
+[ ] PR this document into your community group directory in kubernetes/community (example: sig-architecture/) 
+[ ] by March 8th, 2021
+[ ] titled: annual-report-YEAR.md
+[ ] are there any responses that you’d like to share privately first? steering-private@kubernetes.io or tag your liaison in for discussion. 
+
+# Special Interest Groups:
+(do not copy if you are responding for a working group)
+
+
+# Instructions for this section:
+[ ] each response should have supporting documentation, linked KEPs, graphs, data
+[ ] err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+Operational
+
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate?have a CONTRIBUTING.md file?
+Yes / Yes - README.md is up to date, CONTRIBUTING.md is still accurate but could use some updates.  
+
+All subprojects correctly mapped and listed in sigs.yaml?
+Yes
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+Meetings have decent attendance but few active participants. 
+Meeting notes are used during the meetings are up to date. 
+Recordings are up to date but few views 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+Owners files are up to date. Existing subprojects are mostly in maintenance mode. We would like to try starting one or two subprojects to gather more interest / new participation in SIG-Windows.
+Same question as above but for working groups.
+N/A
```

Comment:
How does the group get updates from the subproject owners? are there any new subprojects in the last year? and then the owners file question, too.